### PR TITLE
Remove explicit widgets activation.

### PR DESCRIPTION
**Pull request by yakutovicha**
It is enough to install widgets - jupyter picks them up automatically.

### CODE REVIEW

**Comment by yakutovicha** [comment]
Tested in a locally build container.

File: Dockerfile [hunk 1]
```diff
@@ -71,40 +71,23 @@ RUN python -m ipykernel install
...
-# Install and enable ipytree.
-RUN /usr/bin/pip3 install ipytree==0.1.8 && \
-  /usr/local/bin/jupyter nbextension install --py --user ipytree && \
-  /usr/local/bin/jupyter nbextension enable --py --user ipytree
+# Install widgets for enabling them in Jupyter.
+RUN /usr/bin/pip3 install bqplot==0.12.25 \
+    ipytree==0.1.8 \
+    ipywidgets-extended==1.0.5  \
+    nglview==2.7.7 \
+    widget-periodictable==2.1.5
```
Comment (csadorf) [inline Dockerfile:90]:
```suggestion
RUN /usr/bin/pip3 install \
    bqplot==0.12.25 \
    ipytree==0.1.8 \
    ipywidgets-extended==1.0.5  \
    nglview==2.7.7 \
    widget-periodictable==2.1.5 \
    && /usr/bin/pip3 cache purge
```

**Review by csadorf — DISMISSED** [review]
Minor suggestion for improved docker command, but otherwise good.

**Comment by yakutovicha** [comment]
I am going to assume that the PR has been approved :)

**Pull request merged**